docs/tutorials/using-lego-dimensions-toy-pad.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Using the LEGO Dimensions Toy Pad
+subject: Hardware - Other
+author: "@JorgePe"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The [LEGO Dimensions game](http://www.lego.com/en-us/dimensions/support) makes
+use of NFC technology to allow some iteraction between physical LEGO creations
+(characters, vehicles and gadgets) and the videogame.
+
+The LEGO Toy Pad is in fact just a custom USB triple NFC reader that can read
+NFC tags (not only the LEGO Toy Tags but also several other tags including those
+used in other games like Disney Infinity) and change the color of the RGB light
+associated to each reader.
+
+## Credits
+
+Since the beginning of 2015 several people have been reverse engineering the
+LEGO Toy Pad.
+This tutorial is strongly based on @woodenphone [work](https://github.com/woodenphone/lego_dimensions_protocol) available at GitHub.
+
+
+## Requirements
+
+You will need:
+
+* a proper LEGO Dimensions Toy Pad
+* a Mindstorms EV3
+* an available USB port
+* python and pysub
+ 
+Several people reported a difference between PS3/PS4/Wii devices and Xbox so
+this tutorial most probaly will not work with the Xbox type.
+
+You don't really need a Mindstorms EV3 as this tutorial can be used on almost any
+recent linux system (like my Ubuntu laptop or my Raspberry Pi). If using ev3dev,
+just be sure to use a recent version - this tutorial was tested with kernel
+`4.4.15-13-ev3dev-ev3`.
+
+Of course, you need an available USB port so if you're already using the Mindstorms
+EV3 with an USB Wi-Fi dongle you will also need an USB Hub.
+
+After you connect the LEGO Toy Pad it should be recognized as an HID device:
+
+    robot@ev3dev:~# dmesg
+    ...
+    usb 2-1.2: new full-speed USB device number 8 using ehci-pci
+    usb 2-1.2: New USB device found, idVendor=0e6f, idProduct=0241
+    usb 2-1.2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+    usb 2-1.2: Product: LEGO READER V2.10
+    usb 2-1.2: Manufacturer: PDP LIMITED. 
+    usb 2-1.2: SerialNumber: P.D.P.000000
+    hid-generic 0003:0E6F:0241.0006: hiddev0,hidraw3: USB HID v1.00 Device
+    [PDP LIMITED.  LEGO READER V2.10] on usb-0000:00:1d.0-1.2/input0
+
+You can also check with lsusb:
+
+    robot@ev3dev:~# lsusb
+    ...
+    Bus 002 Device 008: ID 0e6f:0241 Logic3
+    ...
+
+
+You also need **python** and **pyusb**. Most linux distributions already
+include python as default so you probably only need to install the pyusb library
+with:
+
+    sudo pip install pyusb
+
+but you may need to install pip first:
+
+    sudo apt-get update
+    sudo apt-get install python-pip
+
+
+
+## Initializing
+
+The following script will check for the presence of a LEGO Toy Pad and initializes
+it, turning the middle (round) pad red for one second. It should be invoked with
+`sudo` or root privileges.
```

Comment:
SUBSYSTEM=="usb", ATTR{idVendor}=="0e6f", ATTR{idProduct}=="0241", MODE="0666"
works but only in
/lib/udev/rules.d/99-dimensions.rules
instead of
/etc/udev/rules.d/99-lego-dimensions.conf

Also sudo service udev restart is required (or reboot).